### PR TITLE
chore: visual refresh on menu and menu-item component

### PR DESCRIPTION
**Pull request by khamudom**
# Pull Request

## 📖 Description

before
image

after
image

### General

- [ ] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

### CODE REVIEW

File: packages/web-components/fast-components/src/menu-item/menu-item.styles.ts [hunk 1]
```diff
@@ -44,95 +46,95 @@ export const menuItemStyles: (
...
 
     :host(.indent-0) {
-        grid-template-columns: auto 1fr minmax(42px, auto);
+        grid-template-columns: auto 1fr minmax(32px, auto);
     }
+
     :host(.indent-0) .content {
-        grid-column: 1;
-        grid-row: 1;
+      grid-column: 1;
```
Comment (bheston) [inline packages/web-components/fast-components/src/menu-item/menu-item.styles.ts:69]: Indentation.

File: packages/web-components/fast-components/src/menu-item/menu-item.styles.ts [hunk 2]
```diff
@@ -190,19 +170,14 @@ export const menuItemStyles: (
     :host([role="menuitemcheckbox"]) .input-container,
     :host([role="menuitemradio"]) .input-container {
         display: grid;
-        margin-inline-end: 10px;
     }
 
     :host([aria-haspopup="menu"]) .content,
     :host([role="menuitemcheckbox"]) .content,
-    :host([role="menuitemradio"]) .content {
+    :host([role='menuitemradio']) .content {
```
Comment (bheston) [inline packages/web-components/fast-components/src/menu-item/menu-item.styles.ts:177]:
```suggestion
    :host([role="menuitemradio"]) .content {
```

File: packages/web-components/fast-components/src/menu/menu.styles.ts [hunk 3]
```diff
@@ -28,26 +30,25 @@ export const menuStyles: (
 ) =>
     css`
         ${display("block")} :host {
-            --elevation: 11;
-            background: ${neutralLayerFloating};
+            background: ${fillColor};
             border: calc(${strokeWidth} * 1px) solid transparent;
-            ${elevation}
+            border-radius: ${layerCornerRadius};
```
Comment (bheston) [inline packages/web-components/fast-components/src/menu/menu.styles.ts:35]:
```suggestion
            border-radius: calc(${layerCornerRadius} * 1px);
```

**Review by bheston — APPROVED** [review]

File: packages/web-components/fast-components/src/menu-item/menu-item.styles.ts (hunk 1 above)
Comment (khamudom) [reply packages/web-components/fast-components/src/menu-item/menu-item.styles.ts:69]: updated

File: packages/web-components/fast-components/src/menu/menu.styles.ts (hunk 3 above)
Comment (khamudom) [reply packages/web-components/fast-components/src/menu/menu.styles.ts:35]: updated

File: packages/web-components/fast-components/src/menu-item/menu-item.styles.ts (hunk 2 above)
Comment (khamudom) [reply packages/web-components/fast-components/src/menu-item/menu-item.styles.ts:177]: I believe we use double quotes in FAST

**Review by SethDonohue — APPROVED** [review]

**Pull request merged**